src/i18n/en-US/articles/prepareTrbConsultMeeting.ts

```diff
@@ -0,0 +1,59 @@
+const prepareTrbConsultMeeting = {
+  title: 'Prepare for your TRB consult meeting',
+  tag: 'Technical Review Board',
+  description:
+    'Learn what to expect during and after the meeting, including what to prepare for, and what to bring.',
+  what: {
+    title: 'What is the Technical Review Board?',
+    description:
+      'The CMS Technical Review Board (TRB) is a technical assistance resource for project teams across the agency at all stages of their system’s life cycle. It offers consultations and reviews on an ongoing or one-off basis, allowing project teams to consult with a cross-functional team of technical advisors. It also provides guidance to project teams on adhering to CMS technical standards and leveraging existing technologies.'
+  },
+  whatToExpect: {
+    title: 'What to expect at the meeting',
+    description:
+      'TRB consultation meetings are there to help the TRB understand what your problem or idea is, and details surrounding any proposed solution.  You can expect the TRB and any additional Subject Matter Experts (SMEs) to ask many questions about your system, problem, and solution'
```

Comment:
A couple of typos here - there should only be one space in between sentences and a period after the last sentence.